tests/test_slurm.py

```diff
@@ -59,43 +58,21 @@ def setUpClass(cls):
     def test_sbatch_command(self):
         self.assertTrue(osp.isdir(TestSlurm.CAMPAIGN_PATH))
         for tag in ['uc2']:
-            hpcb_f = osp.join(
-                TestSlurm.CAMPAIGN_PATH,
-                TestSlurm.driver.node,
-                tag, 'hpcbench.yaml'
-            )
-            with open(hpcb_f) as f:
-                hpcb = yaml.safe_load(f)
-            self.assertEqual(hpcb['jobid'], 12345)
-            sbatch = hpcb['sbatch']
-            sbatch_f = osp.join(
-                TestSlurm.CAMPAIGN_PATH,
-                TestSlurm.driver.node,
-                tag, sbatch
-            )
+            root = ReportNode(TestSlurm.CAMPAIGN_PATH)
```

Comment:
very nice! I didn't even think of using this directly here :)

Reply:
I didn't want to write additional unit-tests :)